### PR TITLE
Add ptemcee

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds a recipe for [`ptemcee`](https://github.com/willvousden/ptemcee), including manual packaging of LICENSE file, which is in the repo, but not in the distribution.

**I am not a developer, or maintainer, of `ptemcee`.**

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@conda-forge/help-python, this recipe is ready for review, I hope; thanks.

File: recipes/ptemcee/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (beckermr) [inline recipes/ptemcee/LICENSE]: I checked pypi and the license file is in the download. You can delete it here.

File: recipes/ptemcee/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  requires:
+    - nose
+  commands:
+    - python -m nose --verbose "ptemcee"
+
+about:
+  home: https://github.com/willvousden/ptemcee
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (beckermr) [inline recipes/ptemcee/meta.yaml]: Change this simply to `'LICENSE'`.

**Comment by beckermr** [comment]
Also you don't have to be a developer or maintainer of a package to maintain the conda-forge feedstock! This contribution is great!

**Comment by duncanmmacleod** [comment]
@beckermr, I have removed the license file, and the build passes locally, which is fine. However, the `LICENSE` file is _not_ in the pypi tarball, at least not as far as I can see. How is conda getting to it?

```shell
$ wget -q -O- https://pypi.io/packages/source/p/ptemcee/ptemcee-1.0.0.tar.gz | tar -xvz
x ptemcee-1.0.0/
x ptemcee-1.0.0/PKG-INFO
x ptemcee-1.0.0/ptemcee/
x ptemcee-1.0.0/ptemcee/__init__.py
x ptemcee-1.0.0/ptemcee/interruptible_pool.py
x ptemcee-1.0.0/ptemcee/mpi_pool.py
x ptemcee-1.0.0/ptemcee/sampler.py
x ptemcee-1.0.0/ptemcee/tests.py
x ptemcee-1.0.0/ptemcee/util.py
x ptemcee-1.0.0/ptemcee.egg-info/
x ptemcee-1.0.0/ptemcee.egg-info/dependency_links.txt
x ptemcee-1.0.0/ptemcee.egg-info/PKG-INFO
x ptemcee-1.0.0/ptemcee.egg-info/requires.txt
x ptemcee-1.0.0/ptemcee.egg-info/SOURCES.txt
x ptemcee-1.0.0/ptemcee.egg-info/top_level.txt
x ptemcee-1.0.0/README.rst
x ptemcee-1.0.0/setup.cfg
x ptemcee-1.0.0/setup.py
```

Is it magic?

**Comment by beckermr** [comment]
My sincerest apologies! I clicked the download link on pypi, but one of them gives you a full download of the github repo and the other is the pypi tarball. You are indeed correct!

Sorry for the trouble!

Happy to merge once tests pass!

**Comment by duncanmmacleod** [comment]
@beckermr, no problem, I am just confused as to how my local build appeared to pass without it, which I presume was either me going crazy, or some weird caching artefact. Anyways, builds looks like they are passing again, which is fine.

**Comment by beckermr** [comment]
IDK either on the local build. :/

**Pull request merged**